godspeed/states/world.py

```diff
@@ -192,7 +192,7 @@ def update(self, event_info):
             if dir == "left":
                 pos_x = 29
             else:
-                pos_x = SCREEN_SIZE[0] - (spike_height + 29)
+                pos_x = SCREEN_SIZE[0] - (spike_height + 39)
```

Comment:
Completely agree here 💯 . But unfortunately I think changing this would come under #6 
Right now the arbitrary values are because the platform itself has arbitrary values!